presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2839,6 +2820,22 @@ protected void testStorePartitionWithStatistics(
         }
     }
 
+    protected Partition createDummyPartition(Table table, String partitionName)
+    {
+        return Partition.builder()
+                .setDatabaseName(table.getDatabaseName())
+                .setTableName(table.getTableName())
+                .setColumns(table.getDataColumns())
+                .setValues(toPartitionValues(partitionName))
+                .withStorage(storage -> storage
+                        .setStorageFormat(fromHiveStorageFormat(HiveStorageFormat.ORC))
+                        .setLocation(partitionTargetPath(new SchemaTableName(table.getDatabaseName(), table.getTableName()), partitionName)))
+                .setParameters(ImmutableMap.of(
+                        PRESTO_VERSION_NAME, "testversion",
+                        PRESTO_QUERY_ID_NAME, "20180101_123456_00001_12345"))
```

Comment:
Maybe make the last part have some letters, since that's more common, e.g., `x1y2z`